platform/mellanox/sdk-src/applibs/Makefile

```diff
@@ -0,0 +1,30 @@
+.ONESHELL:
+SHELL = /bin/bash
+
+MAIN_TARGET = applibs_1.mlnx.$(MLNX_SDK_DEB_VERSION)_amd64.deb
+DERIVED_TARGETS = applibs-dev_1.mlnx.$(MLNX_SDK_DEB_VERSION)_amd64.deb
+PACKAGE_NAME = applibs
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# get sources
+	rm -rf $(PACKAGE_NAME)-$(MLNX_SDK_VERSION)-$(MLNX_SDK_ISSU_VERSION)
+
+	wget -c $(MLNX_SDK_SOURCE_BASE_URL)/$(PACKAGE_NAME)-$(MLNX_SDK_VERSION)-$(MLNX_SDK_ISSU_VERSION).tar.gz -O - | tar -xz
+
+	# build
+	pushd $(PACKAGE_NAME)-$(MLNX_SDK_VERSION)-$(MLNX_SDK_ISSU_VERSION)
+
+	sed -i "s/sx-libnl/$(LIBNL3)/g" ./debian/control
+	sed -i "s/-Werror//g" ./configure.in
+
+	if [ -f autogen.sh ]; then
```

Comment:
use -x to test is safer here.

Reply:
A few similar tests in other Makefiles. Use this conversation to track all.

Reply:
Lets have it as different PR againts master, since in master I am also using '-f'. Then it can be cherry-picked to 1811.